dts/msm8916/msm8916-mtp.dts

```diff
@@ -39,4 +39,13 @@
 		compatible = "asus,z010d", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r69339_720p_video:1:none";
 	};
+
+	lenovo-phab {
+		qcom,msm-id = <206 0>;
+		qcom,board-id = <8 0>;
+		
+		model = "Lenovo PHAB";
+		compatible = "lenovo,phab", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt35590_720p_video:1:none";
```

Comment:
Is there really nothing better in the cmdline to match against? Otherwise, why not define the list of panels and use `lk2nd,match-panel;`

Reply:
here's android cmdline: `sched_enable_hmp=1 androidboot.hardware=qcom msm_rtb.filter=0x237 ehci-hcd.park=3 androidboot.bootdevice=7824900.sdhci lpm_levels.sleep_disabled=1 androidboot.emmc=true androidboot.serialno=7be0c63a androidboot.baseband=msm mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt35590_720p_video:1:none`
is there something better to match against?

Reply:
See be73ef2bc8bd7ef2fb8207f319ec85757b06cf0f for example.